feat(SPSTRAT-482): add pre-push for marketplacesvm (#750)
redhat-appstudio/release-service-catalog: pipelines/push-disk-images-to-marketplaces/README.md

| verify_ec_task_bundle           | The location of the bundle containing the verify-enterprise-contract task                              | No       | -                                                         |
| taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                  | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
| taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                         | No       | -                                                         |

## Changes in 0.1.1

> * Allow `marketplacesvm-push-disk-images` task o run on pre-push mode.

Reviewer: ```suggestion
* Allow `marketplacesvm-push-disk-images` task to run in pre-push mode.
```

Author: Fixed
